src/calculateRentalCost.js

```diff
@@ -1,10 +1,18 @@
 /**
- * @param {number} days
+ * @param {number}
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sumOfDays = 40 * days;
+
+  if (days >= 7) {
+    sumOfDays -= 50;
```

Comment:
Use constants for the minimum number of days required to receive a discount and for the discount amounts themselves. This will make the code more maintainable and the values easier to change if needed.